.travis.yml

```diff
@@ -169,6 +174,39 @@ matrix:
       name: "CMake blinky example - debug (NRF52840_DK)"
       env: NAME=cmake_test EXAMPLE_NAME=mbed-os-example-blinky TARGET_NAME=NRF52840_DK PROFILE=debug
 
+    - <<: *cmake-build-test
+      name: "CMake kvstore example (K64F)"
+      env: NAME=cmake_test EXAMPLE_NAME=mbed-os-example-kvstore TARGET_NAME=K64F
+
+    - <<: *cmake-build-test
+      name: "CMake ble example (NRF52840_DK)"
+      env: NAME=cmake_test EXAMPLE_NAME=mbed-os-example-ble TARGET_NAME=NRF52840_DK SUBEXAMPLE_NAME=BLE_Button
+
+    - <<: *cmake-build-test
+      name: "CMake cellular example (WIO_3G)"
+      env: NAME=cmake_test EXAMPLE_NAME=mbed-os-example-cellular TARGET_NAME=WIO_3G
+
+    - <<: *cmake-build-test
+      name: "CMake devicekey example (K66F)"
+      env: NAME=cmake_test EXAMPLE_NAME=mbed-os-example-devicekey TARGET_NAME=K66F
+
+    - <<: *cmake-build-test
+      name: "CMake lorawan example (K66F)"
+      env: NAME=cmake_test EXAMPLE_NAME=mbed-os-example-lorawan TARGET_NAME=K66F
+
+    - <<: *cmake-build-test
+      name: "CMake crypto example (K64F)"
+      env: NAME=cmake_test EXAMPLE_NAME=mbed-os-example-mbed-crypto TARGET_NAME=K64F SUBEXAMPLE_NAME=getting-started
+ 
+    - <<: *cmake-build-test
+      name: "CMake nfc example (DISCO_L475VG_IOT01A)"
+      env: NAME=cmake_test EXAMPLE_NAME=mbed-os-example-nfc TARGET_NAME=DISCO_L475VG_IOT01A SUBEXAMPLE_NAME=NFC_EEPROM
```

Comment:
We need to make sure the EEPROM driver is deployed. It is located in `mbed-os-example-nfc/NFC_EEPROM/source/target/TARGET_M24SR`. Otherwise `m24sr_driver.cpp` cannot be found.

Reply:
This [force-push](https://github.com/ARMmbed/mbed-os/compare/9fe1b713ebd7cbe2273be14d28502b1a81c757fe..c961df3b4f415c3a0e51918e4bcd475d2ff21ee8) adds `mbedtools checkout` to check out Mbed program library dependencies at the revision.
This should also check out the missing library where `m24sr_driver.cpp` is located.